src/components/NavBar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+function NavBar() {
+  return (
+    <footer className="navbar fixed-bottom navbar-light bg-light">
+      <nav className="nav nav-fill w-100">
+        <a className="nav-link" href="#month">
```

Comment:
@Jasonsizemore24 Looks good overall! We'll need to update these link tags to use an onClick attribute or something similar instead of having them link to HTML elements. Probably the click handler should modify a global context that indicates the current view.